### DISCUSSION TITLE
more visibility for managed projects when is open or created

### DISCUSSION THREAD

**Discussion by catafest** [godotengine/godot-proposals#8767] [Editor]
### Describe the project you are working on

I wanted more visibility for managed project folders when I selected them.

![image](https://github.com/godotengine/godot-proposals/assets/2210118/ff89302f-c5f8-46ca-9a37-4202d9a6f0da)

### Describe the problem or limitation you are having in your project

I have many projects from old versions : 3, 4.1, 4.2, and now 4.3, and others from GitHub and the web.
I spent a lot of time to figure out where is and what is the project with 
 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be great if I could give a custom color or icon based on my managed projects.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

This can be done even when project is created and see when I try to open or maybe another custom idea the 

### If this enhancement will not be used often, can it be worked around with a few lines of script?

maybe not because is from Godot GUI

### Is there a reason why this should be core and not an add-on in the asset library?

 because is from Godot GUI

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

> It would be great if I could give a custom color or icon based on my managed projects.

You can already specify project icons in the Project Settings, but you need to open the project in the editor first. If you don't want to open the editor, replace the default `icon.svg` file or create it if it doesn't exist (`icon.png` in Godot 3.x).
